Sources/Views/ALKFormCell.swift

```diff
@@ -147,8 +147,8 @@ class ALKFormCell: ALKChatBaseCell<ALKMessageViewModel>, UITextFieldDelegate, UI
 
     private func setUpTableView() {
         itemListView.backgroundColor = .white
-        itemListView.estimatedRowHeight = 40
-        itemListView.estimatedSectionHeaderHeight = 40
+        itemListView.estimatedRowHeight = 50
```

Comment:
What's the impact of this on iPhone 11 or 11 max?
Add before and after images of those as well. Also, check form buttons that contain text of more than one line

Reply:
Checked in iphone11 &  iPhone 11 Pro Max.adding Screenshots here. Also checked multiline button as well
![iPhone 11 Pro Max  Before](https://user-images.githubusercontent.com/61688116/184634498-f0965eb5-5c0a-4f9f-9fe0-8eea64760532.png)

![iPhone 11 Pro max - After](https://user-images.githubusercontent.com/61688116/184634499-b78e99fb-ccf0-41a2-8aa8-26100dbdf9e9.png)

![iPhone 11 (15 2) Before](https://user-images.githubusercontent.com/61688116/184634485-f59759bf-fd15-40f8-b6ce-469b8e35253f.png)
![iphone 11 (15 2)- After](https://user-images.githubusercontent.com/61688116/184634495-0e22048b-d855-4eec-8ea1-2c4edad2f1d9.png)
.